common/source/docs/common-kogger-sonar.rst

```diff
@@ -0,0 +1,70 @@
+.. _common-kogger-sonar:
+
+=======================
+KOGGER Underwater Sonar
+=======================
+
+
+The `KOGGER Sonar <https://kogger.tech/product/sonar-2d-enhanced>`__ is a compact and low-cost underwater echosounder module with UART interface, 5-14Volts Power Supply Voltage.
+
+.. image:: ../../../images/kogger-sonar.png
+    :target: ../_images/kogger-sonar.png
+
+   
+Recommended Hardware
+--------------------
+- CP210x USB->UART converter and USB OTG cable to configure the module by Android smartphone.
+
+Configuring the echosounder module
+----------------------------------
+The module should be configured before it is connected to the Ardupilot. You need the Android device and GUI installed for configuring the KOGGER Sonar module.
+The GUI is opensource `Kogger Android <https://github.com/koggertech/Kogger-Android/>`__.
+The APK can be downloaded from here: `Kogger Android Releases <https://github.com/koggertech/Kogger-Android/releases/>`__.
```

Comment:
@aproof can you let us know why the APK isn't in the store? You likely have a good reason, but it's bad practice to direct users to APKs, so I'm just wondering the reason.

Reply:
You're right! Now we do not have so many resources to do everything in the best possible way. We'll make it. I mean the app store. However, our users already want to get simple operation and visualization of work. Probably should remove this link. What do you think?

Reply:
Understood. I encourage you to do the work to put it in the store as it gives confidence to users and credibility for you.

Reply:
I absolutely agree with you! We’ll do it soon.